andela/travel_tool_back
# #162192488 Use Alpine Images

#### What does this PR do?
- It uses alpine images instead of carbon images or plain images
- It uses custom image for CircleCI and pulls it from gcr.io

#### Description of Task to be completed?
Currently, we are using carbon images which take almost 600 MBs of space and use outdated version of Node. Change the image to alpine which uses version 10.13.0 of Node.

#### How should this be manually tested?
1. After pulling this branch, run make start, you would see something like this:
```
===> Building required docker images

Building app
Step 1/9 : FROM node:10.13.0-alpine
```

#### Any background context you want to provide?
You might want to delete the previous images to save on space, you can do that using this command:
```
docker images  -q | xargs -I ARGS docker rmi -f ARGS
```

Please note though that the above command would remove ALL the images that are not being used.

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162192488
https://www.pivotaltracker.com/story/show/162192485

#### Screenshots (if appropriate)
<img width="332" alt="screen shot 2018-11-26 at 15 58 08" src="https://user-images.githubusercontent.com/18000724/49015412-18e64c80-f194-11e8-93f7-e9d4a704d367.png">

#### Questions:
N/A
